### ISSUE TITLE
Expected switch device class (in Home Assistant)

### ISSUE THREAD

**Issue by Thecrafterja** [Koenkk/zigbee2mqtt#21477]
### What happened?

Probably since the update to home assistant 2024.2.2, window open of my Bosch thermostat is not available anymore. Developers at HA think it is related to zigbee2mqtt.
Corresponding HA core issue: https://github.com/home-assistant/core/issues/110856
One other person expierences the same problem.

### How to reproduce it (minimal and precise)

Use a Bosch thermostat II, window open option won‘t be available in HA

### Zigbee2MQTT version

1.35.3-1

### Adapter firmware version

20210708

### Adapter

Sonoff Zigbee Dongle-P

### Setup

Add-on in home assistant os on raspi 4

### Debug log

Error log from home assistant (no error in zogbee2mqtt or mosquitto broker log):
```
Logger: homeassistant.components.mqtt.mixins
Source: components/mqtt/mixins.py:270
Integration: MQTT (documentation, issues)
First occurred: 17:48:13 (1 occurrences)
Last logged: 17:48:13

Error 'expected SwitchDeviceClass or one of 'outlet', 'switch' for dictionary value @ data['device_class']' when processing MQTT discovery message topic: 'homeassistant/switch/0x18fc2600000cdb10/switch_window_open/config', message: '{'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': '{{ value_json.state }}'}, {'topic': 'zigbee2mqtt/Thermostat Jannis/availability', 'value_template': '{{ value_json.state }}'}], 'availability_mode': 'all', 'command_topic': 'zigbee2mqtt/Thermostat Jannis/set/window_open', 'device': {'identifiers': ['zigbee2mqtt_0x18fc2600000cdb10'], 'manufacturer': 'Bosch', 'model': 'Radiator thermostat II (BTH-RA)', 'name': 'Thermostat Jannis', 'sw_version': '3.05.09', 'via_device': 'zigbee2mqtt_bridge_0x00124b002a2f253b'}, 'device_class': 'window', 'object_id': 'thermostat_jannis_window_open', 'origin': {'name': 'Zigbee2MQTT', 'sw_version': '1.35.3', 'support_url': 'https://www.zigbee2mqtt.io'}, 'payload_off': 'OFF', 'payload_on': 'ON', 'state_topic': 'zigbee2mqtt/Thermostat Jannis', 'unique_id': '0x18fc2600000cdb10_switch_window_open_zigbee2mqtt', 'value_template': '{{ value_json.window_open }}', 'platform': 'mqtt'}'
```

**Comment by SteffenKr** [comment]
This issue looks like a duplicate of this: #21208
According to a [comment](https://github.com/Koenkk/zigbee2mqtt/issues/21208#issuecomment-1937911133) it should be fixed in the next release (1.36)

**Comment by NicoDreamzZ** [comment]
It is fixed with the release 1.36 :)

**Comment by Thecrafterja** [comment]
Thanks for your responses. I updated my zigbee2mqtt, and it is working again :)
